app/main.py

```diff
@@ -1,3 +1,76 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, distance):
+        if (isinstance(distance, int)):
```

Comment:
Consider float type

Reply:
You can clarify where inemmono needs to be applied float type?

Reply:
You can clarify where inemmono needs to be applied float type?